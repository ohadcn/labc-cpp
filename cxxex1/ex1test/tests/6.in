37377,24664
76906,88256
9071,46235
13184,16834
-72979,19012
8451,-75219
-37313,4655
-65839,35752
71940,36998
11229,-36669
10496,29254
44430,-81812
51705,-52299
-99065,90424
-56617,11708
-3688,80760
-63629,-99694
69015,-27468
46541,-90712
89366,651
28300,-75094
25432,18076
29561,59594
80918,-71411
-3409,-7854
-8079,34176
48489,-63650
52364,-72717
-88860,-19611
-82294,-45477
19186,-58893
62373,55558
-58587,-41524
55179,87954
-32236,44544
-11396,23154
69450,41126
68319,-73900
-72192,49236
-45311,-48512
68471,46610
85664,16960
82960,-34883
44243,94101
45506,-10962
75713,-35309
30146,-34826
47338,71559
23651,2516
-13398,18504
74150,75206
-58343,43599
-56580,-90024
69699,-28772
59212,-75612
22716,27683
-1913,-64531
71732,-91864
586,15974
29326,-53909
-94989,5038
-62129,35157
70213,12299
-66195,-79048
-85186,20407
39456,16053
-4388,8203
-40348,-33878
18179,56440
37350,-95521
7917,87156
-40749,33094
49714,-69018
41230,-49701
-25955,-29445
-76521,-20943
2682,88440
41303,99984
738,75108
-79065,-57358
-77397,-12519
58695,18216
95684,45436
11427,-59049
-71035,75866
-54570,-36029
63021,4682
97065,12734
-37247,38294
-9878,36799
-91152,13601
42945,11531
-70870,-15753
11514,29868
86445,-40461
72510,-90953
47020,-41706
-45648,42703
-96271,-7132
83654,59784
-31266,56173
-76245,58844
87944,-79181
98668,-49303
59114,-11211
87496,-4949
29480,30440
33671,58610
41776,72275
-84434,-44691
-68187,15166
91447,5922
73460,-54202
75714,4278
38667,-13544
-35938,34490
-57372,-12183
-6667,57662
-64275,-8000
-91642,21928
7879,22943
-83021,-62642
80472,-22261
23057,22247
-49987,38623
4646,-91085
-46212,-3908
-58074,54337
-31021,-82361
-41385,34735
4095,22677
-30776,-26187
37583,62558
-68526,73308
81647,66922
-77675,-10475
-10136,66394
-46028,-29665
-55868,77029
19671,-78765
-57259,51406
57240,-3471
-52503,99167
77956,-56434
43895,-63430
78301,75080
-13664,74615
-51108,-76082
-35739,-92545
24315,-54093
74377,46640
35433,91330
-59878,89405
88754,-15745
-6477,-64487
-67421,36264
86920,89819
59882,61506
88985,37837
-94929,-40031
74407,10462
-64952,-39258
-14924,-16060
-88251,49337
-81516,-36846
95244,19950
36883,-42235
-61632,77006
74260,-72879
88350,-32218
89724,-79072
-95955,76643
-89254,63927
-34762,26821
1764,97399
86790,-96741
7860,-78163
91092,20025
-67134,-97160
-30639,78440
93084,91695
-1611,29967
76549,36757
-65939,50808
-9033,-77590
-81411,-19310
-56662,49724
84423,81174
13651,-50340
7994,42504
-52942,-78128
45763,-17992
70799,36854
-97968,-96336
-33216,98484
82104,86957
90178,-92419
16923,-6185
44338,-21927
-55377,-64696
483,90302
-56916,43821
40026,-72494
-47916,80766
-95745,60078
-49642,78403
9040,96121
-39590,-20162
-39936,-37557
-89408,-46063
88016,19785
-59107,5282
27367,-15094
99097,98794
62979,-56281
61188,-36538
-65979,-95729
34372,1136
58867,13545
-18099,63122
-26377,-40652
41524,9752
-44532,29024
89590,-57379
18556,27271
-76353,6571
-52944,-35459
-88148,74423
-23464,-89052
73216,-60486
81758,-38508
-69935,42868
-34237,-8473
-55997,-75371
-94929,-47007
14840,-94217
12341,83454
15535,-5102
12477,-67786
-35391,-68968
59485,-84655
37603,33630
-93025,-23456
-64859,-16489
-12508,-91643
-49885,96339
96939,-19820
39206,-37298
-1204,10298
-85580,3868
63291,29261
-63260,-97280
-60197,-20635
-75292,52280
11579,-83594
10401,98153
-68249,-51997
31783,-61273
-48364,66924
22238,39128
2370,72353
35467,-691
79623,-98239
-10900,-21582
12060,3520
9375,-97561
59870,46116
-94840,99673
-74520,56958
-20959,-35852
73364,89442
89390,-94886
64534,21172
-56159,-83831
-84815,-6831
-17613,-55356
-34479,44943
-28958,72233
46704,60142
-49349,-41237
90752,-12885
88293,-22290
-66770,93453
-95528,85800
-22501,-16487
77037,-49138
-27045,66427
83066,64578
-12402,-73093
80748,-70128
-52835,-36866
-98394,-60225
-91924,-27352
-87992,-18131
-67210,89749
40633,50631
-23137,28925
-71660,-62817
-50534,-67188
-77018,-45946
-56585,-99981
4916,16370
66446,-84930
-19053,81133
69067,88784
38095,-83769
-48083,39701
-43993,87083
-87652,-31985
-31048,72228
-42237,36674
22858,34626
92688,-21712
-1102,-57847
-61811,21881
-76704,-18396
-51011,28212
25063,42524
43282,33100
23656,12349
21883,88840
-44331,890
28540,11676
-12028,40889
6780,56924
-59795,-8368
-6403,-9847
53348,-86627
68441,-47755
-44474,-93371
74126,-21178
-11766,50204
34123,-86704
-7273,-95506
46396,43473
-56068,-31721
32312,-399
69169,-39148
38367,84230
28830,-54854
-31757,69035
36779,61840
-40813,-9874
75214,54717
-30540,57829
61347,-56414
63741,76670
20879,-75048
89966,-86394
56536,-36549
-15832,-99533
-41181,-83521
-72842,55077
-95580,-7386
39306,-39661
-35151,-92452
-70626,-98373
-3522,-84350
-81158,98781
-2543,-11697
-16302,-41197
-41022,-52562
-64528,6946
99481,52527
20552,56016
-84023,31809
-16427,-25204
48289,-62180
56962,-20202
30434,-3732
40138,95283
3816,-3399
24000,27383
-60660,-30069
26163,36797
-14677,-90139
22689,44301
84389,-14750
-48753,10958
-62223,-1112
-33026,80844
57787,-22364
82730,6075
15456,39691
12962,72979
63048,-46901
68261,-33136
49700,-7740
21336,16130
-10719,-52501
-47074,74604
84450,75616
18905,68838
-39135,-2759
-93115,-74269
23218,-26141
33665,-18996
-21416,16394
14168,21129
83174,54220
-5893,46222
-92682,-37633
40175,-15892
-18283,-38490
-99763,-1913
9009,80253
-27310,-6542
-17043,18684
89386,-29088
43014,96271
-3357,-33768
97219,-69693
-25675,-97109
-26210,15583
24020,-43036
-30198,-81874
-96815,-95791
-92418,43360
-11683,-83612
31959,-84357
14475,-59032
95897,87166
-38485,-94058
5849,-49100
-23146,75953
74260,587
-57816,-1433
-42017,-56402
1458,31774
59181,-74523
15827,56072
70693,-53899
60282,78276
16550,75688
21753,-51492
91332,-63772
-10524,14317
50483,-21919
20259,83421
-43930,97114
59373,-69670
-75211,28646
28897,-17228
-666,-69645
-85455,85604
82922,-42539
41676,53615
3562,-70954
-41021,-79889
-95267,80732
95710,23154
44049,-14815
-62530,-78380
-9645,-15181
32131,46426
-90979,-8497
3845,-66190
47239,-67259
16582,-53428
90186,-41783
-67825,197
15678,940
-46189,-80760
29986,-60121
66441,34720
-79390,62150
-42127,-8251
74425,-77567
13369,-35221
-65659,-54501
38294,-56638
-35908,-57862
-22828,38420
-98031,93754
84992,92155
-48030,44256
-7649,-5262
-54804,-26749
13978,75182
13130,-92493
-63010,-39170
-3253,-78047
-20332,-28829
71476,93037
-36961,-94184
65626,-98667
-50822,56807
-56529,26350
-4774,-54560
-52808,-19783
64685,-73748
51562,84125
20990,96758
84466,62057
71940,-75316
-3347,36019
-87397,93400
57972,-7729
-35430,29447
-87603,-45301
35263,78023
83122,11530
-38082,53682
64969,57144
26212,-87839
64450,-9104
65503,16011
-97891,-86419
-60142,13664
75638,38887
38349,-27710
-25095,50952
-34310,-67123
70313,-42651
-10587,82710
-60863,-75324
-12179,-77742
-36705,49739
75941,-71736
-66029,2152
-59575,25510
93048,33017
68610,22246
46598,-91533
-64090,-50676
47354,1348
-51297,-77741
79389,-58518
82226,49701
-1168,-28362
32410,37969
-76597,47320
60228,-13301
-75853,36168
14963,85208
65409,-17523
10717,-14455
-84507,-20673
-92210,-10820
-85117,70790
38504,62238
-27863,87207
11586,-48474
-71311,-79100
-98774,-45390
-7462,-39274
19668,-56969
35135,-20105
-43181,59283
43152,71782
44490,-64351
81348,82297
21194,23930
88713,28985
-86891,-96405
-226,-21298
-7078,-999
92998,-95493
50527,-78314
-74593,-21158
-96615,-54965
-33343,23054
-84845,28882
30038,-28026
-11836,-99721
70845,59743
-64072,52192
42039,57123
-96790,30751
-86804,43409
34347,-59941
49200,54358
39060,42197
-14045,-83325
63884,-61549
-77393,67269
-16514,-83647
-9678,-1359
45235,47450
70615,-66602
47729,-31452
20231,83657
-79261,-37731
40779,51039
20110,53975
-78464,81546
94035,70736
35903,-39817
12933,-78143
-23141,-96095
-39691,-73445
71174,-29116
-57092,88586
-3385,88144
36035,94319
48631,-89148
-37133,95951
94509,-89305
85310,-37623
61735,5419
43441,83271
-13036,-35436
54007,-77134
24748,-33061
71814,28696
-29156,-67878
55251,69107
3006,25248
-15219,26710
-59520,47905
21029,16201
58757,83896
12151,-19645
94591,-2540
42732,-16586
-97122,-13828
-33315,-83069
50737,47781
66888,-24516
41809,38701
31269,39742
-2088,-86392
-64063,28007
38856,20719
-45283,6426
95713,75746
-77374,-18441
59641,34777
61914,54231
59327,4645
-62355,-10706
90817,-68581
6226,41553
79200,-26887
44126,-51903
-88187,-24606
87839,9725
-10998,-76225
64821,54948
-28417,-80463
-38627,-32704
95283,-88912
48855,54923
-27046,10768
-63757,-67720
42502,-99023
21575,33318
-67604,54890
-98040,38685
-71998,46086
86782,66905
48570,1709
-96282,64661
25484,68539
19608,24157
88077,8070
91453,83359
19158,-32604
-34629,92113
5253,1614
51482,-52246
-70320,-26944
81072,-37924
55035,83032
27850,83037
-43793,-58280
-22970,-95224
43429,-19252
-3474,96002
-50713,43224
20158,64453
78383,11610
-25100,-2459
6095,40271
-83258,-88653
68974,-4687
-40899,98654
-4541,67263
87819,-49507
-22617,15668
-66470,-39321
57388,10560
92545,-72095
18398,-10929
-76093,-5226
-67706,44065
-40774,-89324
82765,-38785
-64694,-11141
28576,79138
-72704,-2451
-25549,-86514
-76708,69910
80749,38200
-79598,85222
80958,81022
-54100,-34566
-81329,-61555
-6660,37069
27516,17247
58932,-13101
-38689,-81843
-2425,71165
-93539,-40029
-39976,35037
-60892,-85591
-67414,-59352
-72105,82968
-89443,35734
21167,58050
20955,-70787
-33840,66855
-5352,-15169
-94701,87988
48989,32815
-67677,7920
46803,-79277
-46834,-55622
91889,-40372
4349,79002
94665,-29454
93411,54340
11195,-78695
-35604,-51159
57039,12653
-93110,-22007
41866,-26949
71938,-63487
-15029,77237
51590,61050
-62859,83913
-31031,83944
-95364,-50775
55411,96525
-64058,86849
75526,57696
-42605,-3975
12036,68590
17330,3521
44520,-98542
-83827,51411
-93460,-14871
51551,78478
-51269,63611
-17196,-99679
24660,-52966
11323,93630
-69023,15959
69944,13478
-60428,-94115
326,42187
63581,57721
-61789,-97295
-46601,-17369
6227,97920
-15911,-50511
-23581,-82282
34618,-72031
23286,10439
18669,-93911
10760,-56671
-46877,-77918
-63042,-88811
-34869,33991
24667,4703
39876,-75007
-53110,-69455
9803,-87810
-66749,-9708
-5179,-33433
-84700,5999
16056,-8281
23718,77764
-53223,-52997
-11798,65446
80182,26051
-91226,60394
-24778,72822
71583,40353
-66099,-76661
72146,73777
75422,46125
-95678,-14776
58315,64663
2605,-46864
-68771,17905
-13776,-25626
36713,-90059
79227,10579
84034,94518
-96886,-8696
20568,38978
-48303,95791
-61111,50370
-36768,-27210
73709,-64623
73657,49130
8591,5068
61443,-33094
-30270,-35952
-52869,-71952
9042,33355
2422,-54245
-29614,-91262
56334,81509
-96745,-13463
-27187,23824
-74485,-48401
-53297,-35596
-98032,-62976
-35717,75677
99492,-62061
-48105,8082
43007,-86662
74988,39826
-22614,-77882
67874,86428
-17437,97386
59272,52949
-93877,-57305
34457,-90622
29232,34359
-39709,54747
85959,-65917
46240,87927
98198,-89477
-9307,97689
75552,42588
5770,18559
55926,-92153
85474,60401
57055,53348
-26082,-60383
-22178,-66811
92566,-88966
-24116,-45888
47502,32206
88471,34883
-85959,1518
68966,-12629
-83466,67163
-2106,7227
-8060,-99466
76904,97711
19093,-40081
32647,4567
-79681,-10299
85004,-5763
29319,89915
-45484,-51027
-99052,-42511
-69826,48450
16784,-81355
-16668,30826
-52748,79388
-81804,-36214
-26361,-56821
71013,65580
-56287,47917
-9621,-10104
-92165,-76975
21552,55244
12726,-66356
76571,69134
-76442,58176
45197,51596
-84336,75371
45,32449
21105,10467
90364,68358
-10146,35649
-67857,63493
78828,30246
-43839,49630
-94749,46540
66615,40176
69565,-11834
-4581,-90620
21810,99079
5603,-27543
57254,-49201
-75948,72919
-73830,-48813
-67544,47275
61654,-77181
15632,-48493
-14443,-25135
42089,-35616
-94890,98251
41103,37451
-28121,7718
-22374,68534
95884,134
77914,-55217
-787,-89395
-55671,-43534
-38595,-31618
56474,-12425
19569,16019
61939,-18778
-34073,-95340
-40182,51484
-20475,28996
-84133,-88276
-45665,-15940
49175,26215
91778,53890
-78163,14750
-45977,26840
-13378,53236
37445,30952
36792,98850
99334,-6735
-86487,-54009
-90716,75453
-45698,-24789
80113,-58791
-73305,86727
70205,69652
25540,51630
-46289,-98197
-22156,72578
55693,-73230
-12672,-90285
53610,73950
62952,-8946
4901,-257
-10096,-68677
20097,3417
-95597,-43530
-94042,-41295
31682,13160
-86,-14534
-114,97209
-44883,-74575
48838,-64082
-72772,53771
-91505,10010
80541,-77088
19726,34150
96863,-90234
-74797,-71148
-63402,-57804
87266,-16216
-27297,91669
40255,-21339
77464,-28064
91822,-22623
-15509,-81204
-98326,-33303
71312,-22399
2615,25629
-68629,-61800
-64361,39001
-38888,55365
-26850,85064
-7780,-74557
41006,28818
94729,-44640
-87399,-32569
-52972,-20055
-26818,51581
51881,-34997
56048,-63629
-89112,57722
-69842,82200
-64677,32773
7829,93784
-1938,-29443
32785,59175
53011,33024
-28673,-54770
58467,39422
1137,53196
94782,-59173
47716,68900
20773,-79103
20480,99743
-14100,-23473
63204,-76122
-65751,93362
-93923,-30428
26134,-59005
90445,51286
11552,23229
10460,-35438
-43747,8876
9792,-85281
-51702,38018
-4996,-56921
78846,-57280
-88022,-382
-36383,32459
-73551,-23393
-91015,89653
485,70324
83014,-66349
66985,-63763
-25353,57430
87523,86199
