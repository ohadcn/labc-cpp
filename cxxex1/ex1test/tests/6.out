result
-99763,-1913
-99065,90424
-99052,-42511
-98031,93754
-97968,-96336
-94840,99673
-63629,-99694
-56585,-99981
-11836,-99721
41303,99984
42502,-99023
65626,-98667
79623,-98239
92998,-95493
98198,-89477
99097,98794
99481,52527
99492,-62061
